test/lib/ansible_test/_internal/core_ci.py

```diff
@@ -597,7 +601,7 @@ def ssh_key_callback(files):  # type: (t.List[t.Tuple[str, str]]) -> None
 
 class InstanceConnection:
     """Container for remote instance status and connection details."""
-    def __init__(self, running, hostname, port, username, password):
+    def __init__(self, running, hostname=None, port=None, username=None, password=None):
```

Comment:
Include PEP 484 type hints to match the changes and remove the old PyCharm type hints below.

```suggestion
    def __init__(self,
                 running,  # type: bool
                 hostname=None,  # type: t.Optional[str]
                 port=None,  # type: t.Optional[int]
                 username=None,  # type: t.Optional[str]
                 password=None,  # type: t.Optional[str]
                 ):  # type: (...) -> None
```